packages/admin-ui/src/common/types.ts

```diff
@@ -1,3 +1,6 @@
+import { Compose } from "@dappnode/dappnodesdk/src/files/compose";
+import { ChainDriver } from "@dappnode/dappnodesdk/src/files/manifest";
```

Comment:
You don't want to import deep from inside a module, this things should be exported from the root file